nixos/tests/docker-tools.nix

```diff
@@ -86,6 +86,54 @@ let
       ];
     };
   };
+
+  imageWithoutSigs = pkgs.dockerTools.buildImage {
+    name = "image-without-sigs";
+    tag = "latest";
+    contents = [
+      pkgs.nix
+      pkgs.hello
```

Comment:
do we need to make pkgs.hello non-deterministic here, in order to always have a non-signed pkg?